tool/src/c/mod.rs

```diff
@@ -38,15 +38,15 @@ pub(crate) fn attr_support() -> BackendAttrSupport {
     a
 }
 
+#[derive(askama::Template)]
+#[template(path = "c/runtime.h.jinja", escape = "none")]
+pub(crate) struct Runtime;
```

Comment:
need access to the c runtime for the java backend so that jextract can process it.